ui/app/AppLayouts/Chat/controls/community/CommunityIntroMessageInput.qml

```diff
@@ -15,11 +15,11 @@ StatusInput {
     label: qsTr("Community introduction and rules")
     charLimit: 1400
 
-    input.multiline: true
-    minimumHeight: 400
-    maximumHeight: 400
+    multiline: true
+    minimumHeight: height
+    maximumHeight: (height - 30)
```

Comment:
Looks incorrect, isn't it a binding loop? :eyes: What's the purpose of binding `minimumHeight`/`maximumHeight` to the current height, it does not make any logical sense to me :thinking:

Reply:
IMO it shouldn't be set (min/max height) at all. Looks like some workaround for a bug in `StatusInput` component.

In StatusInput we have:

```
    QtObject {
        id: internal
        readonly property int inputHeight: statusBaseInput.multiline || (root.minimumHeight > 0) || (root.maximumHeight > 0)?
                                  Math.min(Math.max(statusBaseInput.topPadding + statusBaseInput.bottomPadding, 44,
                                                    root.minimumHeight), root.maximumHeight) : 44
    }
```
So when input is multiline, height is set to maxHeight event if it's set to `0`. So ideally we need `StatusInput` to be fixed.